charts/sdp-prototype/data/sdp-devices.json

```diff
@@ -1,8 +1,8 @@
 {
     "servers": {
-        "SdpMaster": {
+        "SDPMaster": {
```

Comment:
Is there going to be any problem with the devices that references this one? i.e. Does the TM know about this name change?

Reply:
There shouldn't be a problem caused by these changes.